### PR TITLE
Jenkins as a Data Science Platform

### PR DESCRIPTION
**Pull request by andykuszyk**
@michaelbinks @bentwright @vanessavirgo1 @dylancsumner @tomashenco what do you think?

If you click `view` on the markdown file on the PR, you can see the markdown rendered with images. Please post any comments you have on here.

Cheers!

### CODE REVIEW

**Comment by andykuszyk** [comment]
Cheers @michaelbinks. Still a work in progress, but wanted to raise a PR to see the markdown render. I'll be adding some images in due course as well.

File: _posts/2018-06-20-jenkins-as-a-data-science-platform.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Jenkins as a Data Science platform
+author: andykuszyk
```
Comment (andykuszyk) [inline _posts/2018-06-20-jenkins-as-a-data-science-platform.md]: Does this need to be changed to `andy_kuszyk` @michaelbinks ?

Comment (michaelbinks) [reply _posts/2018-06-20-jenkins-as-a-data-science-platform.md]: yes it will, good spot, just add the underscore

**Comment by tomashenco** [comment]
Very well written, I like it! 

The only change I would suggest is at the graphs of accuracy / precision. You can find it at 'Jenkins Data Science Summary' -> groupstackreport image. It should be recall instead of accuracy in that case. It's a bug we found some time ago and now it's properly showing recall / precision as can be seen here: http://dev-ap01:8080/job/scorecard/job/train/ (job 78 for instance).

**Comment by andykuszyk** [comment]
Thanks @tomashenco , I've updated the image

**Pull request merged**